### PR TITLE
Update mongo-cxx-driver to 3.8.1

### PR DESCRIPTION
**Pull request by kkloberdanz**
Specify library name and version:  **mongo-cxx-driver/3.8.1**

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by kkloberdanz** [comment]
Hello 👋 Just following up on this ticket. Is there anything missing from this PR that would block it from being merged? Thank you.

File: recipes/mongo-cxx-driver/all/conanfile.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 class MongoCxxConan(ConanFile):
     name = "mongo-cxx-driver"
     license = "Apache-2.0"
+    version = "3.8.1"
```
Comment (uilianries) [inline recipes/mongo-cxx-driver/all/conanfile.py:16]:
```suggestion
```
Please, do not include the package version as recipe attribute. https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#version

**Comment by uilianries** [comment]
> Hello 👋 Just following up on this ticket. Is there anything missing from this PR that would block it from being merged? Thank you.

Hello @kkloberdanz 👋 Thank you for your first contribution! I see that you already signed the CLA. Could you please ask for access in the issue https://github.com/conan-io/conan-center-index/issues/4? The CI bot will scan you there an add in a list of allowed users. It keeps the CI service safer against spam and other bots (are not you a bot, right? 🤖 hahaha)

**Comment by kkloberdanz** [comment]
Thank you, I made the request here: https://github.com/conan-io/conan-center-index/issues/4#issuecomment-1782971593.

**Comment by kkloberdanz** [comment]
Hello, it appears that my request was approved two weeks ago. Is there anything else that I need to do from my end for this ticket?

**Comment by kkloberdanz** [comment]
Hello, just following up again to see if there's anything else that I need to do in order to get this PR reviewed/merged. Thank you.

**Comment by uilianries** [comment]
@kkloberdanz I just restarted the PR now that's you are in the allowed list of users. Please, take a look in the review process to understand it better: https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jwillikers — APPROVED** [review]

**Comment by kkloberdanz** [comment]
Thank you for the approval. I see from the [documentation](https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#automatic-merges) that you linked that approved PRs are automatically merged every 20 minutes. Is there a reason why this PR has not yet been automatically merged? Is there anything that I need to do to get it merged at this point? Thank you.

**Comment by jwillikers** [comment]
> Thank you for the approval. I see from the [documentation](https://github.com/conan-io/conan-center-index/blob/master/docs/review_process.md#automatic-merges) that you linked that approved PRs are automatically merged every 20 minutes. Is there a reason why this PR has not yet been automatically merged? Is there anything that I need to do to get it merged at this point? Thank you.

PR checks are still pending, but once everything has passed it should be queued up for merge automatically. You can see the queue [here](https://prince-chrismc.github.io/conan-center-index-pending-review/#heavy_check_mark-ready-to-merge).

**Pull request merged**